docs/api/components/schemas.yaml

```
@@ -364,29 +366,20 @@ Role:
         Role names can only have alphanumeric ASCII characters and underscores.
       type: string
     admin_channels:
-      description: The channels that users in the role are able to access for the default collection.
+      description: The channels that users in the role are able to access for the default collection. Use `collection_access.scopename.collectionname.admin_channels` for channels in named collections.
```

New line for each sentence

```suggestion
      description: |-
        The channels that users in the role are able to access for the default collection.
        Use `collection_access.scopename.collectionname.admin_channels` for channels in named collections.
```